lib/flappy_search_bar.dart

```diff
@@ -380,9 +468,16 @@ class _SearchBarState<T> extends State<SearchBar<T>>
                       padding: widget.searchBarStyle.padding,
                       child: Theme(
                         child: TextField(
+                          focusNode: _searchQueryFocusNode,
                           controller: _searchQueryController,
-                          onChanged: _onTextChanged,
+                          onChanged:
+                              widget.searchOnlyOnSubmit ? null : _onTextChanged,
+                          onSubmitted:
+                              widget.searchOnlyOnSubmit ? _onTextChanged : null,
                           style: widget.textStyle,
+                          autocorrect: widget.enableSuggestions,
+                          enableSuggestions: widget.enableSuggestions,
+                          textInputAction: TextInputAction.unspecified,
```

Comment:
I like the idea of implementing a `textInputAction`  but why couldn't we give the opportunity to the user to change it itself ?

Reply:
At this point since this PR is already sufficiently huge, I believe abstracting the `textInputAction` for user customisation may be appropriate for a separate PR.